components/batch-system/src/scheduler.rs

```diff
@@ -44,9 +44,9 @@ where
             Priority::Low => &self.low_sender,
         };
 
-        match sender.send(FsmTypes::Normal(fsm), None) {
+        match sender.send(FsmTypes::Normal((fsm, Instant::now_coarse())), None) {
```

Comment:
The precision of now_coarse is typically ms level and I think it may mislead someone.

Reply:
I think `ms` is enough here